src/frontend/Komponenter/Behandling/Vilkårresultat/ResultatVisning.tsx

```diff
@@ -1,49 +1,81 @@
 import React from 'react';
 import { IVurdering, Vilkårsresultat } from '../Inngangsvilkår/vilkår';
-import { Normaltekst, Undertittel } from 'nav-frontend-typografi';
+import { Normaltekst } from 'nav-frontend-typografi';
 import { mapVilkårtypeTilResultat, summerVilkårsresultat } from './utils';
 import styled from 'styled-components';
-import { FlexDiv } from '../../Oppgavebenk/OppgaveFiltrering';
 import { VilkårsresultatIkon } from '../../../Felles/Ikoner/VilkårsresultatIkon';
+import { Label } from '@navikt/ds-react';
+
+const Ikontekst = styled(Normaltekst)`
+    margin-left: 0.25rem;
+`;
+
+const BoldTekst = styled(Label)`
+    margin-right: 1rem;
+`;
 
 const Container = styled.div`
-    padding-top: 1rem;
-    padding-left: 1rem;
-    padding-right: 1rem;
+    width: 280px;
 `;
 
-const Ikontekst = styled(Normaltekst)`
-    margin-left: 0.25rem;
+const ResultatIkonOgTekstWrapper = styled.div`
+    display: flex;
+    justify-content: flex-start;
+    margin-bottom: 0.5rem;
+`;
+
+const ResultatGrid = styled.div<{ lesevisning?: boolean }>`
+    display: grid;
+    grid-template-area: vilkårstype vilkårsoppsummering;
+    grid-template-columns: 7rem 10.5rem;
+    grid-gap: 1rem;
 `;
 
 export const resultatTilTekst: Record<string, string> = {
-    IKKE_AKTUELL: 'ikke aktuell',
-    IKKE_OPPFYLT: 'ikke oppfylt',
-    IKKE_TATT_STILLING_TIL: 'ikke tatt stilling til',
     OPPFYLT: 'oppfylt',
+    IKKE_TATT_STILLING_TIL: 'ikke vurdert',
+    IKKE_OPPFYLT: 'ikke oppfylt',
+    IKKE_AKTUELL: 'ikke aktuell',
     SKAL_IKKE_VURDERES: 'ikke vurdert',
 };
 
+export const resultatTilTall: Record<string, number> = {
```

Comment:
Burde denne kunne bruke typen til `Vilkårsresultat`?
```suggestion
export const resultatTilTall: Record<Vilkårsresultat, number> = {
```

Reply:
Fikset [f4d970e](https://github.com/navikt/familie-ef-sak-frontend/pull/941/commits/f4d970e4d51f2d99ce0e08e47dba98736f5d4e0e)